### PR TITLE
rename valhalla runtime tests to 'inline'

### PR DESCRIPTION
**Pull request by hseigel**
Please review this tiresome change to rename the directory containing the JVM runtime valhalla tests to 'inlinetypes', and to rename the tests and their contents as needed.

Thanks, Harold

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewers
 * Frederic Parain ([fparain](@fparain) - Committer) ⚠️ Review applies to 0c73cdb2647d75269e7b5076a46978e477a65b70

### Download
`$ git fetch https://git.openjdk.java.net/valhalla pull/122/head:pull/122`
`$ git checkout pull/122`

### CODE REVIEW

**Comment by hseigel** [comment]
Please review this updated pull request that replaces <tab>'s with blanks.
Thanks, Harold

File: test/hotspot/jtreg/runtime/valhalla/inlinetypes/InlineTypeDensity.java [hunk 1]
```diff
@@ -221,20 +221,20 @@ public void ensureArraySizeWin() {
...
 
-        LocalDateTimeValue[] valueArray = new LocalDateTimeValue[arrayLength];
+        LocalDateTimeValue[] inlineArray = new LocalDateTimeValue[arrayLength];
         // CMH: add "isFlatValueArray" to WhiteBox API, to ensure we are correctly account size
 
-        long valueArraySize = WHITE_BOX.getObjectSize(valueArray);
-        System.out.println("Object array and elements: " + objectArraySize + " versus Value Array: " + valueArraySize);
-        Asserts.assertLessThan(valueArraySize, objectArraySize, "Value array accounts for more heap than object array + elements !");
+        long inlineArraySize = WHITE_BOX.getObjectSize(inlineArray);
+        System.out.println("Object array and elements: " + objectArraySize + " versus Value Array: " + inlineArraySize);
```
Comment (fparain) [inline test/hotspot/jtreg/runtime/valhalla/inlinetypes/InlineTypeDensity.java:228]: versus Value Array -> versus Flat Array

File: test/hotspot/jtreg/runtime/valhalla/inlinetypes/InlineTypeDensity.java [hunk 2]
```diff
@@ -221,20 +221,20 @@ public void ensureArraySizeWin() {
...
+        Asserts.assertLessThan(inlineArraySize, objectArraySize, "Inline array accounts for more heap than object array + elements !");
```
Comment (fparain) [inline test/hotspot/jtreg/runtime/valhalla/inlinetypes/InlineTypeDensity.java:229]: Inline array -> Flat array

File: test/hotspot/jtreg/runtime/valhalla/inlinetypes/UninitializedInlineFieldsTest.java [hunk 3]
```diff
@@ -20,42 +20,42 @@
...
 import jdk.test.lib.Asserts;
 
 /*
  * @test
  * @summary Uninitialized inline fields test
  * @library /test/lib
- * @compile -XDemitQtypes -XDenableValueTypes -XDallowWithFieldOperator -XDallowFlattenabilityModifiers Point.java JumboValue.java UninitializedValueFieldsTest.java
- * @run main/othervm -Xint -XX:InlineFieldMaxFlatSize=64 runtime.valhalla.valuetypes.UninitializedValueFieldsTest
- * @run main/othervm -Xcomp -XX:InlineFieldMaxFlatSize=64 runtime.valhalla.valuetypes.UninitializedValueFieldsTest
+ * @compile -XDemitQtypes -XDenableValueTypes -XDallowWithFieldOperator -XDallowFlattenabilityModifiers Point.java JumboInline.java UninitializedInlineFieldsTest.java
```
Comment (fparain) [inline test/hotspot/jtreg/runtime/valhalla/inlinetypes/UninitializedInlineFieldsTest.java:31]: Do we still need -XDemitQtypes and -XDenableValueTypes options?

Comment (hseigel) [reply test/hotspot/jtreg/runtime/valhalla/inlinetypes/UninitializedInlineFieldsTest.java]: The -XDemitQtypes and -XDenableValueTypes options are no longer needed.  I removed them from all the runtime valhalla tests.

**Comment by hseigel** [comment]
Hi Fred,
Thanks for the review.  I made the changes that you suggested.  I also don't know why CircularityTest.java and EmptyInlineTest.java appear as deleted/new instead of renamed,  It happened after I replaced the tabs with spaces.

Harold

**Pull request closed without merging**